Marlin/src/pins/ramps/pins_RAMPS.h

```diff
@@ -78,6 +78,31 @@
   #define SERVO3_PIN                           4
 #endif
 
+//
+// Foam Cutter requirements
+//
+
+#if ENABLED(FOAMCUTTER_XYUV)
+  #ifndef MOSFET_C_PIN
+    #define MOSFET_C_PIN                      -1
+  #endif
+  #if HAS_CUTTER && !defined(SPINDLE_LASER_ENA_PIN) && NUM_SERVOS < 2
+    #define SPINDLE_LASER_PWM_PIN              8  // Hardware PWM
+  #endif
+  #ifndef Z_MIN_PIN
+    #define Z_MIN_PIN                         -1
```

Comment:
@DerAndere1: How do you home Z on foam cutters? I've found some online with Y & Z endstops, but you're unsetting the Z endstop here when one isn't defined and that causes compiling to fail for both of our [FoamCutter configs](https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.1.x/config/examples/FoamCutter) with a `Z_MIN_PIN, Z_STOP_PIN, or Z_SPI_SENSORLESS is required for Z axis homing` error.